src/main/java/com/launchdarkly/client/RedisFeatureStore.java

```diff
@@ -50,20 +62,26 @@ public RedisFeatureStore(String host, int port, String prefix, long cacheTimeSec
    * @param uri the URI for the Redis connection
    * @param prefix a namespace prefix for all keys stored in Redis
    * @param cacheTimeSecs an optional timeout for the in-memory cache. If set to 0, no in-memory caching will be performed
+   * @deprecated as of 1.1. Please use the {@link com.launchdarkly.client.RedisFeatureStore#RedisFeatureStore(RedisFeatureStoreBuilder)} constructor for a more
```

Comment:
Now that this ctor is protected, this doc should probably point to the builder's `build` method.